internal/recover/recover.go

```diff
@@ -131,6 +408,7 @@ func CreateRecoveryTarball(filesystem *sys.OS) (string, error) {
 func MaybeUnpackRecoveryTarball(filesystem *sys.OS) error {
 	tarballPath := path.Join(filesystem.StateDir, "recovery_db.tar.gz")
 	unpackDir := path.Join(filesystem.StateDir, "recovery_db")
+	recoveryYamlPath := path.Join(unpackDir, "recovery.yaml")
```

Comment:
Any reason not to remove the `recovery.yaml` when we're done?

Reply:
No :)